microSALT/utils/job_creator.py

```diff
@@ -147,8 +157,11 @@ def blast_subset(self, name, search_string):
 
     if len(file_list) > 1:
       for ref in file_list:
-        ref_nosuf = re.search(r'(\w+(?:\-\w+)*)\.\w+', os.path.basename(ref)).group(1)
-        batchfile.write("# BLAST {} search for {}, {}\n".format(name, self.organism, ref_nosuf))
+        if re.search(r'(\w+(?:\-\w+)*)\.\w+', os.path.basename(ref)) is None:
+          self.logger.error("File {} does not match typical format. Consider deleting and redownloading")
+        else:
+          ref_nosuf = re.search(r'(\w+(?:\-\w+)*)\.\w+', os.path.basename(ref)).group(1)
```

Comment:
duplicated call to `re.search()`it seems. Maybe run it only once and save results to a variable.